src/FlightDisplay/FlightDisplayViewWidgets.qml

```diff
@@ -39,8 +39,11 @@ Item {
     QGCPalette    { id: qgcPal }
 
     function getPreferredInstrumentWidth() {
+        ScreenTools.printStats()
         if(ScreenTools.isMobile) {
             return mainWindow.width * 0.25
+        } else if(ScreenTools.isHugeScreen) {
+            return mainWindow.width * 0.11
         }
         return ScreenTools.defaultFontPixelWidth * 30
```

Comment:
Before it was always some # of pixels * 30.
After it is 11% of screen width if using 4k screen (which has tiny pixels and apparently the same defaultFontPixelWidth as lower res screens).

Reply:
Ah, I see.

Reply:
I was confused that this isn't a mobile case.